<launch>
  <include file="$(find rm_description)/launch/sentry.launch">
    <arg name="load_chassis" value="true"/>
    <arg name="roller_type" value="realistic"/>
    <arg name="paused" value="true"/>
  </include>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
        respawn="false" output="screen">
    <remap from="/joint_states" to="/controller/joint_states" />
  </node>
  <rosparam file="$(find sentry_chassis_controller)/config/controller.yaml" command="load" ns="controller"/>
  <rosparam file="$(find sentry_chassis_controller)/config/teleop_twist_keyboard.yaml" command="load" ns="teleop_twist_keyboard"/>
  <!-- load the controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
        output="screen" args="
        controller/sentry_chassis_controller
        controller/joint_state_controller    
  "/>
  <!-- load teleop_twist_keyboard node -->
  <node name="teleop_twist_keyboard" pkg="sentry_chassis_controller" type="teleop_twist_keyboard" launch-prefix="gnome-terminal -e"/>
</launch>
